packages/workbox-webpack-plugin/src/lib/utils/read-file.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Use the `compiler.inputFileSystem._readFile` method instead of `fs.readFile`,
+ * `readFile` is configured to use `compiler.inputFileSystem._readFile` during
+ * the run phase of the webpack compilation lifecycle by passing the function
+ * to the `setReadFile` function.
+ */
+let readFileFn;
```

Comment:
I'm not crazy about this pattern but I can see why you've done it.

Reply:
Yeah, me either. This is just incase we can't use `fs` or `fse` in the webpack compilation. Is there a better pattern for this you would suggest?